cpp/CMakeLists.txt

```diff
@@ -1105,7 +1105,7 @@ if(CUDF_BUILD_STREAMS_TEST_UTIL)
       ${_tgt} PRIVATE "$<BUILD_INTERFACE:$<$<COMPILE_LANGUAGE:CXX>:${CUDF_CXX_FLAGS}>>"
     )
     target_include_directories(${_tgt} PRIVATE "$<BUILD_INTERFACE:${CUDF_SOURCE_DIR}/include>")
-    target_link_libraries(${_tgt} PUBLIC CUDA::cudart rmm::rmm)
+    target_link_libraries(${_tgt} PUBLIC CUDA::cudart rmm::rmm cudf::cudf)
```

Comment:
This is fine, but FWIW I believe it should be sufficient to link to rmm::rmm_logger and rmm::rmm_logger_impl instead of cudf based on the error you're seeing

Reply:
I'm not even sure where the logger is being used here. And why the error only shows up in a Debug build.

Reply:
My guess would be that [this](https://github.com/rapidsai/rmm/blob/3cad8fed150c3905962ca849b11bfc12fadd2517/include/rmm/detail/logging_assert.hpp#L41) is triggering